editor.planx.uk/src/ui/editor/ModalFooter.tsx

```diff
@@ -20,32 +22,47 @@ export const ModalFooter = <T extends BaseNodeData>({
   showInternalNotes = true,
   showTags = true,
   disabled,
-}: Props<T>) => (
-  <>
-    {showMoreInformation && (
-      <MoreInformation
-        changeField={formik.handleChange}
-        definitionImg={formik.values.definitionImg}
-        howMeasured={formik.values.howMeasured}
-        policyRef={formik.values.policyRef}
-        info={formik.values.info}
-        disabled={disabled}
-      />
-    )}
-    {showInternalNotes && (
-      <InternalNotes
-        name="notes"
-        onChange={formik.handleChange}
-        value={formik.values.notes}
-        disabled={disabled}
-      />
-    )}
-    {showTags && (
-      <ComponentTagSelect
-        value={formik.values.tags}
-        onChange={(value) => formik.setFieldValue("tags", value)}
-        disabled={disabled}
-      />
-    )}
-  </>
-);
+}: Props<T>) => {
+  const isTemplate = useStore.getState().isTemplate;
+
+  return (
+    <>
+      {showMoreInformation && (
+        <MoreInformation
+          changeField={formik.handleChange}
+          definitionImg={formik.values.definitionImg}
+          howMeasured={formik.values.howMeasured}
+          policyRef={formik.values.policyRef}
+          info={formik.values.info}
+          disabled={disabled}
+        />
+      )}
+      {showInternalNotes && (
+        <InternalNotes
+          name="notes"
+          onChange={formik.handleChange}
+          value={formik.values.notes}
+          disabled={disabled}
+        />
+      )}
+      {showTags && (
+        <ComponentTagSelect
+          value={formik.values.tags}
+          onChange={(value) => formik.setFieldValue("tags", value)}
+          disabled={disabled}
+        />
+      )}
+      {isTemplate && (
+        <TemplatedNodeConfiguration
+          formik={formik}
+          isTemplatedNode={formik.values.isTemplatedNode}
+          templatedNodeInstructions={formik.values.templatedNodeInstructions}
+          areTemplatedNodeInstructionsRequired={
+            formik.values.areTemplatedNodeInstructionsRequired
+          }
+          disabled={disabled}
+        />
+      )}
```

Comment:
This is a main change: anytime the `ModalFooter` is rendered within the editor modal of a source template flow, it'll automatically have another final section !